qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -92,11 +94,10 @@
             remove_fs_source = yes
             fs_target = 'myfs'
             fs_driver_props = {"queue-size": 1024}
-            mem = 4096
```

Comment:
Hi @xiagao 

I am confuse about why we remove this limitation and then add a limitation for Win10.i386 only,would you please share more about that?Thanks.
In my understanding, previous limitation will cover current PR's limitation.